### ISSUE TITLE
Automate test of tv-train

### ISSUE THREAD

**Issue by MarvinTeichmann** [TensorVision/TensorVision#22]
The following commands should be executed:
- tv-train --hypes examples/cifar10_vgg8.json --debug
- tv-train --hypes examples/cifar10_minimal.json --debug
- tv-train --hypes examples/cifar10_cnn.json --debug

The following result is expected:
- output/debug/output.log is touched (i.e. has a very recent timestamp)
- the std:out contains a line `Filling queue with 5000 CIFAR images before starting to train. This will take a few minutes.`
- The std:out contains a line `2016-05-06 10:49:13,147 INFO Step 0/100000: loss = 4.88 ( 0.031 sec (per Batch); 4094.6 examples/sec)` after reasonable time

Is there any way to nicely automate this test? So far I am doing this check by hand  before every commit.

**Comment by MarvinTeichmann** [comment]
@MartinThoma Do you want to have a look at this? Otherwise I will tackle it within the next days.

**Comment by MartinThoma** [comment]
@MarvinTeichmann Not really. This will probably be quite a bit of work (and I have an exam on Tuesday)
